src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -59,6 +59,190 @@ public static void FlushPayload()
 ";
 
         [Fact]
+        public void HelpersInstrumentation()
```

Comment:
probably not. I was using this to (manually) verify the extra execution trace caused by instrumented flush method in other tests are correct.